book/toc.md

```diff
@@ -52,3 +52,32 @@ Instead edit toc_template.md
 
 * [This book on GitHub](https://github.com/InnerSourceCommons/InnerSourcePatterns)
 * [InnerSource Commons](http://innersourcecommons.org)
+
+## Translation<a id="p"></a>
+
+* Japanese
+  * [イントロダクション](../translation/japanese/introduction.md)
+  * [パターンの一覧](../translation/japanese/explore-patterns.md)
+  * [この本へのコントリビューション](../translation/japanese/contribute.md)
+  * パターン
```

Comment:
I think this was intended, but the order of the Table of Contents is sorted alphabetically according to the names of the Japanese titles, not according to the order of the original English titles.
If they were arranged according to the English titles, it would be easier to compare them with the English originals.
However, if it is aligned along the Japanese title, it is easier to find a pattern that starts with katakana. However, I did not find it that much easier to find patterns that begin with kanji.
This may be a point of contention, so I would like to comment here for your reference and would like to hear other people's opinions.

Reply:
Thanks for pointing that out! I couldn't agree more.

This ToC is actually auto-generated. Please refer to this [code](https://github.com/yuhattor/InnerSourcePatterns-Japanese/blob/japanese-translation-v1/book/generate_toc.rb). The English version just takes the contents of the following folder.

- `./patterns/2-structured/*.md`
- `./patterns/2-structured/project-setup/*.md`
- `./patterns/3-validated/*.md`

The Japanese version follows this. For Japanese, I think it is possible to sort by specific rules by adding a program:)

Reply:
@hirotakatoya, @kanazawazawa, @johtani,
I'd like to hear any opinions. What kind of order do you think would be best for users? 

I think it is easy to find the order indicated in the InnerSource Mind Map, but since the ToC is automatically generated, it is a bit difficult to check for omissions. However, I think it is not impossible.
But on the other hand, for users with no knowledge of InnerSource, it may be difficult to find them in any order,  so a random order may be fine as well.

Reply:
@yuhattor At first, I think it's okay to use the same order as the original. Why don't you think about it again when it increases and becomes difficult to see? I don't have a strong belief in this idea, so it seems likely that it doesn't matter which one.

Reply:
@yuhattor , Thank you for your contribution. My opinion on this issue is that, in principle, we should follow Gitbook's standard way.

If there is a problem with Gitbook's standard way (maybe it is not firm enough but I found [one example](https://xiaochen-su.gitbook.io/gitbook_multilanguage_template/en/gitbook_multilanguage_template)), then I think the problem should be fixed by Gitbook. This problem will be essentially fixed, if Gitbook updated.

Of course, it would be nice if there was some action taken as a temporary measure until then, but I don't think it is an essential problem. As @kanazawazawa says, I think it is one option as a temporary measure to keep the table of contents in the original order.

The other thing I'd like to add to this issue is how the PDF feature works. I checked [the prototype](https://george-15.gitbook.io/innersource-patterns-in-japanese/), but unlike the original page, the PDF functionality is not present in the menu and I could not verify it. Thank you.